[ADD] payment_providers: Nuvei (#11157)
odoo/documentation: content/applications/finance/payment_providers/nuvei.rst

#. Go to :guilabel:`Settings` in the left sidebar, select :guilabel:`My Integration Settings`, then
   select your payment page in the :guilabel:`Website` dropdown menu.
#. Copy the :guilabel:`Merchant ID` and :guilabel:`Site ID` values and save them for :ref:`later
   <payment_providers/nuvei/odoo>`.
#. Click :guilabel:`Reveal`, then copy the :guilabel:`Secret Key` and save it for :ref:`later
   <payment_providers/nuvei/odoo>`.

.. tip::
   Nuvei supports additional functionality relating to error handling from the same page as above.
   For example, the following options are available for handling failed transactions: Enable
   :guilabel:`User Trap` to redirect users back to the payment page and allow them to attempt
   another deposit or :guilabel:`Decline Recovery` to display a popup suggesting alternative or
   similar payment methods.

.. _payment_providers/nuvei/odoo:

Configuration on Odoo
=====================

#. :ref:`Navigate to the payment provider Nuvei <payment_providers/add_new>` and change its state
   to :guilabel:`Enabled`.
#. Fill in the :guilabel:`Merchant Identifier`, :guilabel:`Site Identifier`, and :guilabel:`Secret

>    Key` fields with the information saved at the step :ref:`payment_providers/nuvei/dashboard`.

Reviewer: I don't know if it's super relevant, but I'd swap these two steps, as switching to the "enabled" state marks required the fields to fill at step 2.

Author: Ordering swapped!

#. Configure the rest of the options to your liking.

.. tip::
   You can also test Nuvei using the :ref:`test mode <payment_providers/test-mode>` and your Sandbox
   Dashboard values.

.. _payment_providers/nuvei/services:

> Providers By Country
> ====================

Reviewer 2: ```suggestion
Payment methods
===============
```
I think you meant payment methods instead of providers? 
(And we use [sentence case in headings](https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html))

> Most Nuvei payment methods are **region specific**. The payment methods supported by a country are
> listed below:

Reviewer 2: ```suggestion
Most Nuvei payment methods are **region-specific**. The supported payment methods for each country are listed below:
```
